members/S001194.yaml

```diff
@@ -1,76 +1,65 @@
-    - wait:
-        - value: 2
-    - find:
-        - selector: select#actions
-    - click_on:
-        - selector: select#actions
-    - find:
-        - selector: option[value='opinion']
-    - click_on:
-        - selector: option[value='opinion']
-    - wait:
-        - value: 2
-    - find:
-        - selector: form#contactForm input#fname
+    - visit: "https://www.schatz.senate.gov/contact/share-your-thoughts/form"
+    - find: 
+        - selector: "#input-3012F9F8-E1FF-15AC-780D-51E059038981"
     - fill_in:
         - name: First Name
-          selector: "#fname"
+          selector: "#input-3012F9F8-E1FF-15AC-780D-51E059038981"
           value: $NAME_FIRST
           required: true
         - name: Last Name
-          selector: "#lname"
+          selector: "#input-3019E679-D896-A600-EA99-2AFF6192202B"
           value: $NAME_LAST
           required: true
         - name: Street Address
-          selector: "#mailing_streetAddress1"
+          selector: "#input-3028D193-DFD8-C4F5-9EA9-18136ADF4E5D"
           value: $ADDRESS_STREET
           required: true
         - name: Street Address2
-          selector: "#mailing_streetAddress2"
+          selector: "#input-3028D200-C0AF-E2EA-97B7-AC1A28A47AA7"
           value: $ADDRESS_STREET_2
           required: false
         - name: City
-          selector: "#mailing_city"
+          selector: "#input-3028D249-0458-5091-56F5-8F95636CE606"
           value: $ADDRESS_CITY
           required: true
         - name: ZipCode
-          selector: "#mailing_zipCode"
+          selector: "#input-3028D2F9-A2B6-52F8-2ACC-1060ED58A2B7"
           value: $ADDRESS_ZIP5
           required: true
         - name: Phone number
-          selector: "#home_phone_number"
+          selector: "#input-302BB54B-04DD-6592-A3CA-950B84CBEB21"
           value: $PHONE
           required: false
-        - name: email
+        - name: Email
           selector: "#email"
           value: $EMAIL
           required: true
-        - name: verify_email
-          selector: "#verify_email"
+        - name: Verify email
+          selector: "#input-3034610B-FC23-1553-E7D8-920486264617"
           value: $EMAIL
           required: true
         - name: message
-          selector: "#message"
+          selector: "#input-30403D9F-9D8B-E1D8-02F8-5D7C17B516E7"
           value: $MESSAGE
           required: true
     - javascript:
         - name: char replace
           values: ["$MESSAGE"]
-          selectors: ["#message"]
+          selectors: ["#input-30403D9F-9D8B-E1D8-02F8-5D7C17B516E7"]
           commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/(%2[0C])/g, '');}"]
           required: true
     - select:
-        - name: Your Prefix
-          selector: "#salutation"
+        - name: Prefix
+          selector: "#input-3014F950-07E6-309C-4FC0-3FE72DF6CBFB"
           value: $NAME_PREFIX
           required: true
           options:
+            "None": "None"
```

Comment:
Since `None` is an option now, can you try something where we initially select "None" then have the actual prefix step run afterwards and have it NOT marked as required? this is one of those fields that would be great to not require if possible